src/modules/45-dashboards/components/WidgetLibrary/WidgetLibrary.tsx

```diff
@@ -155,12 +157,7 @@ export const WizardLibrary: React.FC<{ onClose: () => void }> = ({ onClose }): J
           )}
 
           <Layout.Horizontal className={css.footerActions}>
-            <Button
-              text="Add to Dashboard"
-              intent="primary"
-              disabled={!selectedLook}
-              onClick={() => addWidgetToDashboard()}
-            />
+            <Button text="Add to Dashboard" intent="primary" disabled={!selectedLook} onClick={addWidgetToDashboard} />
```

Comment:
You'll need a `ButtonVariation` here as well.
```suggestion
            <Button
              text="Add to Dashboard"
              intent="primary"
              variation={ButtonVariation.PRIMARY}
              disabled={!selectedLook}
              onClick={addWidgetToDashboard}
            />
```